### PR TITLE
Qualification tool: Parsing Execs to get the ExecInfo #2

### PR DESCRIPTION
**Pull request by nartal1**
This fixes https://github.com/NVIDIA/spark-rapids/issues/5361. It is a follow on PR of https://github.com/NVIDIA/spark-rapids/pull/5420.

Execs supported in this PR: 
- GlobalLimitExec
- LocalLimitExec
- GenerateExec
- CartesianProductExec
- SortExec

Have added eventlog for testing GlobalLimit and LocalLimitExec. These execs are not present in final physical plan if `collect` is called. So the eventlog is generated using `show` (df.limit(10).show).  For rest of the execs, we create eventlogs on thr fly in the test itself.

### CODE REVIEW

File: tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala [hunk 1]
```diff
@@ -82,6 +85,11 @@ class SQLPlanParserSuite extends FunSuite with BeforeAndAfterEach with Logging {
           assert(projects.forall(_.isSupported == true))
           assert(projects.forall(_.children.isEmpty))
           assert(projects.forall(_.duration.isEmpty))
+          val sorts = allChildren.filter(_.exec == "SortExec")
```
Comment (tgravescs) [inline tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala:88]: is it called SortExec or Sort?  The tests above for FilterExec and SortExec actually have a bug in them.
Add a check to make sure the sorts.size is 1

**Review by tgravescs — COMMENTED** [review]
why do we have a global_local_limit_eventlog, I thought those execs were already in the nds_q86_test eventlog?

File: tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala (hunk 1 above)
Comment (nartal1) [reply tools/src/test/scala/com/nvidia/spark/rapids/tool/planparser/SqlPlanParserSuite.scala]: Thanks for catching this. Updated the test. There are 3 Sorts in the the plan.

**Comment by nartal1** [comment]
build

**Comment by nartal1** [comment]
build

**Pull request merged**